cmd/notary/tuf.go

```diff
@@ -123,7 +123,12 @@ func tufInit(cmd *cobra.Command, args []string) {
 		fatalf(err.Error())
 	}
 
-	keysMap := nRepo.KeyStoreManager.RootKeyStore().ListKeys()
+	var keysMap map[string]string
```

Comment:
Initializing like this raises a panic when you try and assign into it because memory hasn't been allocated. At this point keysMap is a nil pointer (map and slice variables are implicitly pointers to the underlying data, which is why they can take the `nil` value). Needs to be initialized as `keysMap := make(map[string]string)`